src/main/java/repository/OrderRepository.java

```diff
@@ -1,48 +1,135 @@
 package repository;
 
+import exception.CarRentalException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import mapper.Mapper;
 import models.order.Order;
 
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Timestamp;
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
 import java.util.UUID;
 
+@Slf4j
+@RequiredArgsConstructor
 public class OrderRepository {
-    public boolean create(Order order) {
-        //INSERT INTO "order"(client_id, car_id, from, to, total_price) VALUES ('?', '?', ?, ?, ?);
-        return true;
+    private final Connection connection;
+
+    public Order insert(Order order) {
+        String INSERT = "INSERT INTO \"order\"(client_id, car_id, from, to, total_price) VALUES (?, ?, ?, ?, ?);";
+        try (PreparedStatement statement = connection.prepareStatement(INSERT)) {
+            statement.setObject(1, order.getClientId());
+            statement.setObject(2, order.getCarId());
+            statement.setTimestamp(3, getTimestamp(order.getFrom()));
+            statement.setTimestamp(4, getTimestamp(order.getTo()));
+            statement.setDouble(5, order.getTotalPrice());
+            statement.execute();
+            return order;
+        } catch (SQLException exception) {
+            log.error("Can not process statement", exception);
+            throw new CarRentalException(exception.getMessage());
+        }
     }
 
-    public Order getById(UUID id) {
-        //SELECT * FROM "order" WHERE id = '?' AND status
-        return null;
+
+    public List<Order> getAll() {
+        String GET_ALL = "SELECT * FROM \"order\" WHERE status";
```

Comment:
Do you need these "" here? Or it can be removed?

Reply:
I need them. 
Without them I get error.